app/helpers/users_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UsersHelper
+end
```

Comment:
Normally we don't introduce something that we don't use. 
Thanks!

Reply:
Okay bhai. Removing it. Thanks.